src/Network/Request.php

```diff
@@ -134,7 +134,7 @@ class Request implements ArrayAccess
         'json' => ['accept' => ['application/json'], 'param' => '_ext', 'value' => 'json'],
         'xml' => ['accept' => ['application/xml', 'text/xml'], 'param' => '_ext', 'value' => 'xml'],
     ];
-
+    
```

Comment:
Please make sure your IDE trims right. There should be no whitespace noise here.